### PR TITLE
Add tsvector index to comments.body_markdown

### PR DESCRIPTION
**Pull request by atsmith813**
## What type of PR is this? (check all applicable)
- [x] Optimization

## Description
This PR follows https://github.com/forem/forem/pull/13401 and adds a tsvector index on `comments.body_markdown` to help speed up comments search.

## Related Tickets & Documents
https://github.com/forem/forem/pull/13401
https://ui.honeycomb.io/thepracticaldev/datasets/rails/result/oQgEzyaXWqK/a/vtBoa2nkXx4/comments

## QA Instructions, Screenshots, Recordings
1. Run `rails db:migrate` and see the index is added.
2. Run `rails db:rollback STEP=1` and see the index is removed.

## Added tests?
- [x] No, and this is why: this is a DB migration - no tests required.

## [Forem core team only] How will this change be communicated?
- [x] I will share this change internally with the appropriate teams

## [optional] Are there any post deployment tasks we need to perform?
I'll sync with SRE to monitor the results of this change.

## [optional] What gif best describes this PR or how it makes you feel?

faster_jurassic_park_gif

### CODE REVIEW

File: db/migrate/20210421190438_add_body_markdown_tsvector_index_to_users.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class AddBodyMarkdownTsvectorIndexToUsers < ActiveRecord::Migration[6.1]
```
Comment (djuber) [inline db/migrate/20210421190438_add_body_markdown_tsvector_index_to_users.rb:1]: the index is being added to Comments and not users, probably need to update the name.

Comment (atsmith813) [reply db/migrate/20210421190438_add_body_markdown_tsvector_index_to_users.rb:1]: Classic copy/pasta, great catch - updated!

**Review by djuber — APPROVED** [review]
migrations had the desired effect up and down

**Review by citizen428 — APPROVED** [review]

**Review by rhymes — APPROVED** [review]
LGTM!

**Pull request merged**